src/support/notfound.js

```diff
@@ -99,3 +78,44 @@ export const send404Report = async ({
   // eslint-disable-next-line no-await-in-loop
   return slackClient.postMessage({ ...slackContext, blocks, unfurl_links: false });
 };
+
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+function isWithinLast7Days(date) {
```

Comment:
the method name should not contain a concrete vaule - can this be changed to `isWithinDays(data, numDays)` ?